tests/e2e/set-ffi-env-e2e

```diff
@@ -156,13 +157,18 @@ install_qm_rpms() {
   dnf config-manager --set-enabled crb
 
   local release_id
-  release_id=$(grep ^ID= /etc/os-release | cut -d = -f 2)
+  release_id=$(grep -oP '(?<=^ID=)\w+' /etc/os-release)
+  local version_id
+  version_id=$(grep -oP '(?<=^VERSION_ID=)\w+' /etc/os-release)
+  if [[ "$release_id" == "centos" ]]; then
+    release_id=epel
+  fi
 
   if [[ -n "${USE_QM_COPR}" ]]; then
       if [[ "${USE_QM_COPR}" != "release" ]]; then
-          dnf copr enable -y @centos-automotive-sig/bluechi-snapshot centos-stream-9
-          dnf copr enable -y "${USE_QM_COPR}" centos-stream-9
-      elif [[ "${USE_QM_COPR}" == "release" && "${release_id}" =~ "centos" ]]; then
+          dnf copr enable -y @centos-automotive-sig/bluechi-snapshot "$release_id-$version_id-$(arch)"
```

Comment:
Question, isn't now part of .packit.yaml? (do we need it here too)?  Maybe we can remove in one place?

Reply:
this one, for installing PR copr rpm for build, 
There are many options for manual tests and debug
see the command, https://github.com/containers/qm/pull/652/files#diff-fd76bc4afd6e791b352e18d8bcda536f540475342be7ef1f72737960369d3343R271

Reply:
No, I meant, in theory this enable should be automatic these days with .packit.yaml. Right?

Reply:
Yes, but i would like to propose it in a different PR once ros or libkrun fedora tests to be introduced

Reply:
@Yarboa so should we merge this one and later you address the duplication?